cmd/protoc-gen-go-grpc/go.mod

```diff
@@ -0,0 +1,8 @@
+module google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+go 1.9
+
+require (
+	github.com/golang/protobuf v1.4.0-rc.4
```

Comment:
Should this minimum be updated, or this this on purpose?

Reply:
Updated.